File: 프로그래머스/0/181943. 문자열 겹쳐쓰기/문자열 겹쳐쓰기.js
function solution(my_string, overwrite_string, s) {
    let answer = '';
    const myStrLeng = my_string.length;
    const overwriteStrLeng = overwrite_string.length;
    const restStrLeng = myStrLeng - overwriteStrLeng - s;
    answer += my_string.slice(0, s);
    answer += overwrite_string;
    if (restStrLeng === 0) return answer;
    return answer += my_string.slice(-restStrLeng);
}

File: 프로그래머스/0/181943. 문자열 겹쳐쓰기/문자열 겹쳐쓰기.py
def solution(my_string, overwrite_string, s):
    answer = ''
    l = len(overwrite_string)
    
    front = my_string[:s]
    middle = overwrite_string
    back = my_string[s + l:]
    
    answer = front + middle + back
    
    return answer